environment.yml

```diff
@@ -0,0 +1,10 @@
+name: pylhe
+channels:
+  - conda-forge
+dependencies:
+  - hist
```

Comment:
Super minor comment but it would make sense to give at least lower limits to these dependencies as we do for the package itself.